### DISCUSSION TITLE
budibase remote connection to database

### DISCUSSION THREAD

**Discussion by ctorres-dev** [Budibase/budibase#12138] [General]
hello,

I have a problem with budibase, it is already configured and working, but I need to connect to remote databases, I have created users with permissions to the ips and different configurations to mysql to allow connections from any interface but I still can not connect to external databases from budibase .... I must enable some option so that the budibase can connect because from terminal I can do it.

**Comment by melohagan** [comment]
Hey @ctorres-dev 

Are you self-hosted or cloud? Do you need to configure SSL?
